impl.go

```diff
@@ -208,6 +212,10 @@ func (p Pkg) fullType(e ast.Expr) string {
 	return p.gofmt(e)
 }
 
+func (p Pkg) inRecvPkg() bool {
```

Comment:
Please just inline this, as it is used in only the one place.

Reply:
I think using inRecvPkg() is more readable and extensible.

Reply:
Respectfully, I disagree.

Reply:
If you're going to keep this as a separate function, it needs docs.

Reply:
> Respectfully, I disagree.
Sorry, didn't notice your reply. Since you disagree, I will delete it.